index.html

```diff
@@ -1,190 +1,194 @@
+              <select id="type_of_search" name="typeOfSearch" class="input">
+                <option value="issue">Issues</option>
+                <option value="pr">Pull Request</option>
+              </select>
+
+              <input
+                type="text"
+                placeholder="User/Organization"
+                list="users_history"
+                id="input_user"
+                name="user"
+                class="input calendary_filter__repository_name"
+              />
+              <input
+                type="text"
+                placeholder="Repository name"
+                list="repositories_history"
+                id="input_repository"
+                name="repository"
+                class="input calendary_filter__repository_name"
+              />
+
+              <div class="align-r mt-3">
+                <button id="btn_next" class="btn btn-sm">Next</button>
+              </div>
+            </div>
+
+            <div id="tab_2" class="box d-none">
+              <select id="input_condition" name="condition" class="input">
+                <option value="after">After of</option>
+                <option value="before">Before of</option>
+                <option value="equals">Equals to</option>
+              </select>
+
+              <input
+                type="date"
+                name="date"
+                id="input_date"
+                class="calendary_filter__date_search input"
+              />
+
+              <div class="p-relative">
+                <button id="goback" class="btn--back p-absolute mt-6">
+                  <img
+                    src="./src/assets/arrow.png"
+                    style="margin-top: 3px"
+                    width="15px"
+                    height="15px"
+                    alt=""
+                  />
+                </button>
+              </div>
+
+              <div class="div mt-3">
+                <button id="redirectToSearch" class="btn btn-sm">Ir</button>
+                <button id="copySearch" class="btn btn--secondary btn-sm">
+                  Copiar
+                </button>
+              </div>
+            </div>
+            <datalist id="repositories_history">
+              <!-- TODO: ADD repositories history on this datalist -->
+            </datalist>
+            <datalist id="users_history">
+              <!-- TODO: ADD users history on this datalist -->
+            </datalist>
+          </form>
+        </div>
+      </div>
+    </main>
+
+    <script type="module">
+      import {
+        getFromLSOrDefaultValue,
+        validateForm,
+        createSearchFilter,
+        getFormValues,
+      } from "./src/js/utils.js";
+
+      /*
+       * STEPS FORM
+       */
+      const btnNext = document.getElementById("btn_next");
+      const goBack = document.getElementById("goback");
+      const tab1 = document.getElementById("tab_1");
+      const tab2 = document.getElementById("tab_2");
```

Comment:
I was checking and these variables are not used elsewhere, what about putting inside `switchTab` function?